secretsauce/apps/portal/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
Can we register the models we add below here so that they appear in the admin site too?